docs/transparent-validation/supermicro_x11_lga1151_series/theory-of-operation.md

```diff
@@ -0,0 +1,49 @@
+# Supermicro X11 LGA1151 Series theory of operation
+
+## Remote power control using ipmitool
+
+To remote control supermicro you can use ipmitool.
+
+```bash
+ipmitool -H [bmc_ip] -U [bmc_username] -P [bmc_password] chassis power off
+ipmitool -H [bmc_ip] -U [bmc_username] -P [bmc_password] chassis power on
+ipmitool -H [bmc_ip] -U [bmc_username] -P [bmc_password] chassis power cycle
+ipmitool -H [bmc_ip] -U [bmc_username] -P [bmc_password] chassis power reset
+```
+
+## Enabling SOL
+
+This procedure is based on
+[this documentation](https://www.fmad.io/blog/supermicro-serial-kvm).
+
+Follow the steps below to set up a serial connection via ssh:
+
+1. Enable SOL/COM2 console in BIOS. To do that in BIOS go to the `Advanced`
+   section, then select `Serial Port Console Redirection` and set the
+   `SOL/COM2 Console Redirection` option to `Enabled` state.
+
+1. Edit the `/etc/default/grub` on your Linux-based OS file by adding:
+
+    ```bash
+    linux /vmlinuz ro console=ttyS1,115200n8 earlyprintk=serial,ttyS1,115200n8
+    ```
+
+1. Connect with BMC via ssh:
+
+    ```bash
+    ssh ADMIN@<IP>
```

Comment:
It should work via password authentication? It requires a key it appears. And documentation does not mention how to add one.

Reply:
I prepared the documentation on a system with OpenSSH < 8.8, and it worked without any configuration.
https://github.com/Dasharo/docs/pull/585/commits/53c1c25d09798d99eef8d08b88cf1fea0e20f2c4 - I added the missing step

Reply:
@macpijan ping

Reply:
@krystian-hebel any ideas?

Reply:
The updated documentation worked for me.